.travis.yml

```diff
@@ -38,7 +36,6 @@ script:
   - ulimit -n
   - coverage run --omit="test/*" -m unittest --buffer
   - coverage report
-  - if [ "$TRAVIS_PYTHON_VERSION" == '3.5' ]; then cd doc && make html; fi
```

Comment:
I think this should check for `'3.6'` instead to not loose documentation validation.